hbs-deposits.html

```diff
@@ -0,0 +1,185 @@
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+        integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+
+    <style>
+        .required label::after {
+            content: "*";
+            color: red;
+            margin-left: 5px
+        }
+
+        #card-errors {
+            background-color: red;
+            color: white;
+            font-weight: bold;
+        }
+    </style>
+</head>
+
+<body>
+    <div class="container" style="padding-top: 14px; border: 1px solid #ced4da; border-radius: 5px;">
+        <form id="payment-form" action="">
+            <div class="form-group required">
+                <label for="first_name">First Name</label>
+                <input id="first_name" class="form-control" type="text" required/>
+            </div>
+
+            <div class="form-group required">
+                <label for="last_name">Last Name</label>
+                <input id="last_name" class="form-control" type="text" required/>
+            </div>
+
+            <div class="form-group required">
+                <label for="email">Email</label>
+                <input id="email" title="By making a donation, you will receive updates on the Henry Barnard School. You may unsubscribe at any time." class="form-control" type="email" placeholder="name@example.com" required/>
+            </div>
+
+            <div class="form-group required">
+                <label for="phone">Phone</label>
+                <input id="phone" title="We may contact you at this number about your donation and other opportunities to support the Henry Barnard School." class="form-control" type="phone" placeholder="123-345-6789" required/>
+            </div>
+
+            <div class="form-group required">
+                <label for="studend_first_name">Student First Name</label>
+                <input id="student_first_name" class="form-control" type="text" required/>
+            </div>
+
+            <div class="form-group required">
+                <label for="student_last_name">Student Last Name</label>
+                <input id="student_last_name" class="form-control" type="text" required/>
+            </div>
+            
+
+            <div class="form-group required">
+                <label for="name">Credit Card</label>
+                <div id="card-element" style="padding: 10px; border: 1px solid #ced4da; border-radius: 5px; margin-bottom: 30px;">
+                    <!-- Elements will create input elements here -->
+                </div>
+            </div>
+
+            
+            <!-- We'll put the error messages in this element -->
+            <div id="card-errors" role="alert" class="mb-3 pl-2 pt-2 pb-2 pr-2 d-none"></div>
+
+            <div>
+                <p>Your card will be charged $1029</p>
```

Comment:
We know there is a 2.9% "convenience fee" here but is that explained anywhere? may want to add it here